dotCMS/src/main/java/com/dotmarketing/util/Mailer.java

```diff
@@ -293,6 +297,16 @@ public void addAttachment(java.io.File file, String filename) {
 			}
 		}
 	}
+	public void setHeader(String name, String value) {
```

Comment:
if setHeader and addHeader do the same, it could be replaced by `customHeaders.put(headerName, headerValue);`. It seems like the if conditional is not necessary